### PR TITLE
CSP-105-Web-ConfirmRegulatedStandard

### PR DESCRIPTION
**Pull request by raviudari01**

### CODE REVIEW

File: src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/StandardsController.cs
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/StandardsController.cs:84]: What is the purpose of Request.Scheme and Request.Host? Other url seems to be working with out adding these information.

**Review by DevenKShah — CHANGES_REQUESTED** [review]
Half way in review I realised that this has not been merged well, I can see changes from 122 in here

File: src/SFA.DAS.Roatp.CourseManagement.Application/Standard/Commands/UpdateApprovedByRegulator/UpdateApprovedByRegulatorCommandHandler.cs
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.CourseManagement.Application/Standard/Commands/UpdateApprovedByRegulator/UpdateApprovedByRegulatorCommandHandler.cs:30]: similar note to above,, change 'confirm' to 'set approval for '

**Review by MarkFCain — COMMENTED** [review]
I've added a few optional changes to words in logs/exceptions, I'll leave it to you.  I will approve when Deven's issues are resolved

**Review by MarkFCain — APPROVED** [review]

File: src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/StandardsController.cs
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/StandardsController.cs:82]: Missed check if IsRegulatedStandard

**Review by DevenKShah — APPROVED** [review]
Minor suggestions

**Pull request merged**
